src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -15,10 +15,17 @@
 --%>
 <%@ page import="codeu.model.store.basic.ConversationStore" %>
 <%@ page import="codeu.model.store.basic.UserStore" %>
+<%@ page import="codeu.model.store.basic.MessageStore" %>
+<%@ page import="codeu.model.data.Message" %>
 
 <%
   UserStore userStore = UserStore.getInstance();
+  userStore.sort();
```

Comment:
This isnt a bad place to sort, but I think a more a place that would make more sense is right in the "getMostActive..." methods. This way the code explains why we are getting the last element and we can use the methods in whatever context we want.

Reply:
But… if I do it in those, it will run more times the sorting, or I am wrong?